squirrel.pc.in

```diff
@@ -0,0 +1,12 @@
+prefix=@CMAKE_INSTALL_PREFIX@
+libdir=@CMAKE_INSTALL_FULL_LIBDIR@
+includedir=@CMAKE_INSTALL_FULL_INCLUDEDIR@
+
+Name: @PROJECT_NAME@
+Version: @PROJECT_VERSION@
+Description: @PROJECT_DESCRIPTION@
+URL: @PROJECT_HOMEPAGE_URL@
+Requires:
+Provides: squirrel3 = @PROJECT_VERSION@
+Libs: -L${libdir} -lsquirrel3 -lsqstdlib3
```

Comment:
`squirrel3`/`sqstdlib3` exist only when `LONG_OUTPUT_NAMES` is defined for the build, otherwise they are `squirrel`/`sqstdlib`. I guess the library names should be generated/substituted as well.